Documentation/advertising-identifier.md

```diff
@@ -0,0 +1,53 @@
+# Advertising identifier
```

Comment:
can we move this under getting-started-test-app since it is related to that, maybe rename to "Testing with advertising identifiers"